cn_indian_payroll/cn_indian_payroll/page/payroll_configuratio/payroll_configuratio.js

```
@@ -735,590 +32,398 @@ frappe.pages['payroll-configuratio'].on_page_load = function(wrapper) {
                     }).appendTo($('<td>').appendTo(row));
 
-                    var abbrInput = $('<input>', {
+                    const abbrInput = $('<input>', {
                         type: 'text',
                         class: 'form-control',
                         value: item.abbr || "",
                         readonly: !isComponentEditable
                     }).appendTo($('<td>').appendTo(row));
 
-                    var conditionInput = $('<input>', {
+                    let conditionInput = $('<input>', {
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'conditionInput' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)